docs/tutorials/google/reservations.ipynb

```diff
@@ -119,36 +119,12 @@
     }
    ],
    "source": [
-    "def authenticate_user():\n",
-    "    \"\"\"Runs the user through the Colab OAuth process.\n",
-    "    \n",
-    "    Checks for Google Application Default Credentials and runs interactive login \n",
-    "    if the notebook is executed in Colab. In case the notebook is executed in Jupyter notebook\n",
-    "    or other IPython runtimes, no interactive login is provided, it is assumed that the \n",
-    "    `GOOGLE_APPLICATION_CREDENTIALS` env var is set or `gcloud auth application-default login`\n",
-    "    was executed already.\n",
-    "    \n",
-    "    For more information on using Application Default Credentials see \n",
-    "    https://cloud.google.com/docs/authentication/production\n",
-    "    \"\"\"\n",
-    "    in_colab = False\n",
-    "    try:\n",
-    "        from IPython import get_ipython\n",
-    "        in_colab = 'google.colab' in str(get_ipython())\n",
-    "    except: \n",
-    "        # Notebook is not executed within IPython. Assuming external authentication.\n",
-    "        return \n",
-    "\n",
-    "    if in_colab: \n",
-    "        from google.colab import auth      \n",
-    "        print(\"Getting OAuth2 credentials.\")\n",
-    "        print(\"Press enter after entering the verification code.\")\n",
-    "        auth.authenticate_user(clear_output=False)\n",
-    "        print(\"Authentication complete.\")\n",
-    "    else: \n",
-    "        print(\"Notebook is not executed with Colab, assuming Application Default Credentials are setup.\") \n",
+    "# The Google Cloud Project id to use.\n",
+    "project_id = \"\" #@param {type:\"string\"}\n",
+    "processor_id = \"\" #@param {type:\"string\"}\n",
     "\n",
-    "authenticate_user()"
+    "if not get_device_sampler(project_id, processor_id, False):\n",
```

Comment:
This needs to check if signed in; the boolean value of the returned (nonempty) tuple will always be true.

Reply:
what is the third argument used in the call here. Is that a mistake?

Reply:
This whole line was a mistake, it's fixed now.